README.md

```diff
@@ -26,6 +26,8 @@ interface, for more information read the documentation below.
 
 ## Table of Contents
 
+[![Greenkeeper badge](https://badges.greenkeeper.io/Shopify/draggable.svg)](https://greenkeeper.io/)
```

Comment:
Did greenkeeper add this? We already have one at the top of the readme. When can remove it as a follow-up

Reply:
Yeah it did, lol. K im going to drop that commit then fixup into a single commit and merge

Reply:
ugh, when I rebase its show commits that should be there... `git rebase -i HEAD~3` does not give me what I'd expect...